packages/devtools_app/lib/src/screens/memory/shared/widgets/heap_class_view.dart

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+import '../../../../../devtools_app.dart';
+import '../heap/model.dart';
+
+class HeapClassView extends StatelessWidget {
```

Comment:
Does this need to be in its own file? I worry about starting to feel too java-like where every class is in its own file and things start to feel "overly" abstracted. If the concern is that this class is used in multiple places, maybe it makes sense to create a `shared_widgets` file in the memory code where all widgets can go that are shared in multiple panes